src/appkit.cs

```diff
@@ -13344,7 +13344,7 @@ interface NSSliderTouchBarItem
 		NSSliderAccessory MaximumValueAccessory { get; set; }
 
 		[Export ("valueAccessoryWidth")]
-		double ValueAccessoryWidth { get; set; }
+		nfloat ValueAccessoryWidth { get; set; }
```

Comment:
This is a TouchBar API, which IIRC isn't in stable yet so it should be safe to change, but would need to be included with the rest of the TouchBar additions